corehq/apps/domain/templates/domain/admin/add_form_repeater.html

```diff
@@ -1,7 +1,8 @@
-{% extends "settings/bootstrap2/base_template.html" %}
+{% extends "settings/bootstrap3/base_template.html" %}
 {% load i18n %}
 {% load hqstyle_tags %}
 {% load hq_shared_tags %}
+{% load crispy_forms_tags %}
 
 {% block js-inline %} {{ block.super }}
```

Comment:
don't forget to look for css styles in the javascript if things don't look like the original view. in this case `label-important` should be changed to `label-danger`